### DISCUSSION TITLE
1979-05-23-lena-zavaroni-and-music/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#18] [General]
# 1979-05-23-lena-zavaroni-and-music/

This week’s guests are Elaine Stritch, Wayne Sleep and George Chisholm.

https://fanzoflenazavaroni.github.io/1979-05-23-lena-zavaroni-and-music/

**Comment by StephenMolloy1** [comment]
Dougie Squires, the choreographer of the show died on Sunday (21st May 23).

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for the sad news.
